### ISSUE TITLE
QLdds_131.pkg: expired certificate

### ISSUE THREAD

**Issue by commitay** [Marginal/QLdds#2]
```
Package "QLdds_131.pkg":
   Status: signed by a certificate that has since expired
   Certificate Chain:
    1. Developer ID Installer: Jonathan Harris
       SHA1 fingerprint: 46 AE 72 AF E2 BC 26 7A A8 D1 BB FB 4B 59 41 A2 C1 49 68 60
       -----------------------------------------------------------------------------
    2. Developer ID Certification Authority
       SHA1 fingerprint: 3B 16 6C 3B 7D C4 B7 51 C9 FE 2A FA B9 13 56 41 E3 88 E1 86
       -----------------------------------------------------------------------------
    3. Apple Root CA
       SHA1 fingerprint: 61 1E 5B 66 2C 59 3A 08 FF 58 D1 4A E2 24 52 D1 98 DF 6C 60
```

This was initially reported to [Homebrew-Cask](https://github.com/caskroom/homebrew-cask/issues/39727).

**Comment by Marginal** [comment]
Fixed in release 1.32.

**Comment by commitay** [comment]
Thanks!

https://github.com/caskroom/homebrew-cask/pull/39826
